src/pages/Products/Productdetail/Productdetail.js

```diff
@@ -1,8 +1,264 @@
 import React from 'react';
+import './Productdetails.scss';
+
+// 금액 합계
+// 1. 빈 배열 생성
+// 2. 셀렉트박스에서 옵션 선택(클릭)할 때
+// 3. data.price의 금액을 빈 배열로 가져오삼
+// 4. 그리고나서 합계 란에 그 배열 값 띄우기
+// 4. 다른 옵션 선택 할 때마다 data.price금액이 계속 더해짐
+// 배열? number니까 스트링은 안되고.. 배열에 넣는게 맞나?
+
+// 상품 수량 조절
+// 1. 상품 수량조절기 (+)를 클릭하면
+// 2. 수량 숫자에 1씩 더해짐
+// 3. 그리고 맨 아래 total값에도 data.price값이 더해짐
+// 4. (-)버튼 누르면 총 합계에서 data.price값이 빠짐
+
+// 수량 숫자가 1일때는 (-)버튼 비활성화
+// close아이콘 클릭시 영역 한 줄 제거
 
 class Productdetail extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      data: {},
+      emptyList: [],
+      type: [],
+      totalPrice: 0,
+      quantity: 1,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/mockData_detail_page.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(res => {
+        this.setState({ data: res.items });
+      });
+
+    // http://10.58.7.206:8000/options
+
+    fetch('data/mockData_detail_page_second.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(res => {
+        this.setState({ type: res.option });
+      });
+  }
+
+  select = e => {
+    this.setState({
+      emptyList: this.state.emptyList.concat({
+        type: e.target.value,
+        quantity: 1,
+      }),
+      totalPrice: this.state.totalPrice + this.state.data.price,
+    });
+  };
```

Comment:
```suggestion
  select = e => {
    const { emptyList, totalPrice, price } = this.state
    
    this.setState({
      emptyList: [ ...emptyList, { type: e.target.value, quantity: 1 }),
      totalPrice: totalPrice + data.price,
    });
  };
```
- 구조분해 할당 해주시면 조금 더 코드가 깔끔해질 거 같습니다.
- concat과 같은 의미에서 스프레드 연산자를 사용해주시면 조금 더 직관적으로 배열에 새로운 요소를 추가해줄 수 있을거 같아요.